scripts/o3de/ExportScripts/export_standalone_monolithic_project_centric.py

```diff
@@ -48,10 +48,20 @@
 
 # For more information on the available parameters for this script, check the parse_arguments function defined at the bottom of the file
 
+# Account for some windows-specific attributes
+if platform.system().lower()=='windows':
+    EXECUTABLE_EXTENSION = '.exe'
+    ADDITION_BUILD_ARGS = ['--', '/m', '/ nologo']
```

Comment:
Space here too.
```suggestion
    ADDITION_BUILD_ARGS = ['--', '/m', '/nologo']
```

Reply:
Also the name should be ADDITION**AL**_BUILD_ARGS, since it is being used in a verb context.